manual/src/tutorials/objectexamples.etex

```diff
@@ -998,8 +998,8 @@ p = q, p = p;;
 Other generic comparisons such as ("<", "<=", ...) can also be used on
 objects.  The
 relation "<" defines an unspecified but strict ordering on objects.  The
-ordering relationship between two objects is fixed once for all after the
-two objects have been created and it is not affected by mutation of fields.
+ordering relationship between two objects is fixed indefinitely once the
```

Comment:
My non-native speaker feeling is that  `indefinitely` is not a good fit for an unalterable relation.

Reply:
`permanently` might be more appropriate

Reply:
Changed to `permanently`.